libcodechecker/libhandlers/analyze.py

```diff
@@ -174,6 +174,14 @@ def is_ctu_capable():
                                     "compiler on the system is special, e.g. "
                                     "when doing cross-compilation.")
 
+    analyzer_opts.add_argument('--capture-analysis-output',
+                               action='store_true',
+                               default=False,
+                               required=False,
+                               help="Store the (standard and error) output "
+                                    "of the successful analyzer invocations "
+                                    "into the <report-dir>/success directory.")
```

Comment:
Path should be formatted as `'<OUTPUT_DIR>/success'` (note the apostrophes around the path, this is how relative PATHs are mentioned in help). The `metavar` for `-o` is `OUTPUT_DIR`.